src/posts/en/configuring-babel-6-for-node-js.md

```diff
@@ -0,0 +1,164 @@
+<!--
+layout: post
+title: Configuring Babel 6 for Node.js
+date: 2015-12-31T01:23:42.811Z
+comments: true
+published: true
+keywords: JavaScript, Node.js, ES6, babel, transpiler
+description: Tutorial on configuring babel for node.js to get up and running quickly
+categories: ES6, babel, node.js
+authorName: Hannan Ali
+authorLink: https://abdulhannanali.github.io
+authorPicture: //s.gravatar.com/avatar/89e5f7614cb88cd573359a953a09aa6e?s=80
+-->
+Hi! If you are like me, you are tired of writing the same old ES5 JS Code in your node applications.
+
+If yes, you can use the newer features of JavaScript ES6 and ES7 standards in your node applications today. ES6 and ES7 make the JavaScript development a cool breeze. But hey, not every ES6 feature is supported in our beloved [Node.js](https://nodejs.org).
+
+This is where [BabelJS](https://babeljs.io) comes to the rescue. BabelJS is a transpiler for JavaScript which transpiles your ES6 and ES7 code into ES5 and even ES3 code. In simple words it converts it into JavaScript that node.js can run and make you really happy.
```

Comment:
In some parts of the tutorial, you call it "BabelJS" and, in others, "babel". It would be nice to have consistent naming. I think it would be better to always name it "Babel", as that is how it is written in the https://babeljs.io/ site.